src/Storage/Storage/Common/Cmdlet/NewAzureStorageAccountSasToken.cs

```diff
@@ -54,6 +56,15 @@ public class NewAzureStorageAccountSasTokenCommand : StorageCloudBlobCmdletBase
         public override int? ClientTimeoutPerRequest { get; set; }
         public override int? ConcurrentTaskCount { get; set; }
 
+        protected override bool UseTrack2SDK()
```

Comment:
Do you want to rename this function to `UseTrack2Sdk`.

Reply:
fixed,